src/services/redisSessionStorage.ts

```diff
@@ -37,8 +41,6 @@ export default class RedisSessionStorage implements ISessionStorage {
     });
 
     const setWalletToken = new Promise<Either<Error, boolean>>(resolve => {
-      // Set key to hold the string value. If key already holds a value, it is overwritten, regardless of its type.
```

Comment:
unstage this change as it's not in the scope of this PR

Reply:
fixed on 2354a0b